marta_msgs/Position position
float32 speed
string control_mode
float32 altitude
